onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -1079,6 +1075,9 @@ def run_onnxruntime(args, models):
                 if ep not in validation_exemption:
                     try:
                         ort_outputs = inference_ort_and_get_prediction(name, sess, inputs)
+                        #logger.info("COMPARE OUTPUTS Ref, Ort") 
```

Comment:
what are these commented out lines?

Reply:
I was using it to investigate the accuracy issues. I can delete them for this PR.